src/argus_ticket_rt.py

```diff
@@ -21,45 +29,41 @@ class RequestTrackerPlugin(TicketPlugin):
     def import_settings(cls):
         try:
             endpoint, authentication, ticket_information = super().import_settings()
-        except ValueError as e:
-            LOG.exception("Could not import settings for ticket plugin.")
-            raise TicketPluginException(f"Request Tracker: {e}")
+        except TicketSettingsException as e:
+            LOG.exception(e)
+            raise TicketSettingsException(f"Request Tracker: {e}")
 
-        if (
-            "username" not in authentication.keys()
-            or "password" not in authentication.keys()
+        if "token" not in authentication.keys() and (
+            "username" not in authentication.keys() or "password" not in authentication.keys()
         ):
-            LOG.error(
-                "Request Tracker: No username or password can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
+            LOG.exception(
+                "Request Tracker: No authentication details (token or username/password) can be found in the authentication information."
             )
-            raise TicketPluginException(
-                "Request Tracker: No username or password can be found in the authentication information. Please update the setting 'TICKET_AUTHENTICATION_SECRET'."
+            raise TicketSettingsException(
+                "Request Tracker: No authentication details (token or username/password) can be found in the authentication information. Please check and update the setting 'TICKET_AUTHENTICATION_SECRET'."
             )
 
         if "queue" not in ticket_information.keys():
-            LOG.error(
-                "Request Tracker: No queue can be found in the ticket information. Please update the setting 'TICKET_INFORMATION'."
-            )
-            raise TicketPluginException(
-                "Request Tracker: No queue can be found in the ticket information. Please update the setting 'TICKET_INFORMATION'."
+            LOG.exception("Request Tracker: No queue can be found in the ticket information.")
+            raise TicketSettingsException(
+                "Request Tracker: No queue can be found in the ticket information. Please check and update the setting 'TICKET_INFORMATION'."
```

Comment:
See comment for previous exception.

Reply:
In general, there's a lot of exceptions raised and they all have the same pattern: a log, then the log-message is re-used in the exception-message, verbatim. You do not wish to change them pair-wise if anyone objects to what is logged. Maybe better to log the entire exception-message instead of the first half.